Makefile

```diff
@@ -12,23 +12,36 @@ bin/boot.o: src/boot/boot.asm | bin
 		-o $@ \
 		$<
 
-# -O ReleaseFast
-bin/boot_zig.o: src/boot/boot.zig | bin
-	zig build-obj -target x86_64-freestanding-gnu -static -I./src/kernel -mno-red-zone -femit-bin=$@ $<
+PKGS=--pkg-begin zasm ./src/zasm.zig --pkg-end
```

Comment:
Do we call these source files zasm? So they are then imported as `@import("zasm")`?

Reply:
Yeah, we give the source file `./src/zasm.zig` the package name `zasm`. Note, the source file could have been called `hello_world.zig` and we could still assign it to the package name `zasm`.

This is a getto way to handle packages, until the package manager is implemented in Zig (ziglang/zig#943).